src/condor_ce_run

```diff
@@ -117,7 +130,7 @@ def submit_job(job_info):
     if job_info['remote']:
         args += ['-remote', job_info['schedd_name'], '-pool', job_info['collector_name']]
 
-    rc, stdout, stderr = ce.run_command(args)
+    rc, stdout, stderr = run_command_debug(args, debug=job_info['debug'])
```

Comment:
Should check to make sure `job_info['debug']` exists, otherwise this will throw a KeyError. I know it's only called from one place right now where the `debug` key is set, but this is a general-use function that might get used later in other places.